admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -979,6 +980,16 @@ var form = (function() {
       });
   }
 
+  function checkDOMChange() {
+    if($('.attribute_wholesale_price').length > 0 && $('.attribute_priceTE').length > 0) {
+      inputsTimeout = null;
+      $('.attribute_wholesale_price, .attribute_priceTE').each(function(){
+        $(this).val(priceCalculation.normalizePrice($(this).val()));
+      })
+    }else
+      inputsTimeout = setTimeout( checkDOMChange, 100 );
```

Comment:
I think if you use the "**DOMSubtreeModified**" event instead of using "**setTimeout**", it will be better.
Example : `$('element').on('DOMSubtreeModified', function() { ... })`

https://developer.mozilla.org/en-US/docs/Web/Events/DOMSubtreeModified